docroot/modules/humsci/hs_blocks/src/Plugin/Block/SocialMediaBlock.php

```diff
@@ -0,0 +1,386 @@
+<?php
```

Comment:
This block plugin probably would've been easier to build with a custom fieldable custom block type. The link fields, weights, etc would've all existed and only some validation and templating would've been needed.